glib/src/auto/functions.rs

```diff
@@ -16,6 +16,33 @@ pub fn access(filename: impl AsRef<std::path::Path>, mode: i32) -> i32 {
     unsafe { ffi::g_access(filename.as_ref().to_glib_none().0, mode) }
 }
 
+#[doc(alias = "g_assertion_message_cmpint")]
+pub fn assertion_message_cmpint(
```

Comment:
doesn't look useful to have in bindings & the ffi lacks a since annotation

Reply:
Yeah all those assertion/test etc fnctions need to be manually implemented, if it all. But this is unnecessary and has very nice Rust crates for the same purpose.